Cubical/Homotopy/Group/Pi4S3/S3PushoutIso.agda

```diff
@@ -0,0 +1,680 @@
+{-# OPTIONS --safe #-}
+{-
+This file has been split in two due to slow type checking
+combined with insufficient reductions when the
+experimental-lossy-unification flag is included.
+Part 2: Cubical.Homotopy.Group.Pi4S3.S3PushoutIso2
```

Comment:
@Saizan This is a bit weird... Some parts need experimental-lossy-unification to terminate, but some other parts don't typecheck if we have it... Any clue what we can do?

Is it somehow possible to have flags at the level of modules instead of files?